lib/workers/repository/__fixtures__/master-issue_with_8_PR.txt

```diff
@@ -18,6 +19,7 @@ These updates are awaiting their schedule. Click on a checkbox to get an update
 
 These updates are currently rate limited. Click on a checkbox below to force their creation now.
 
+ - [ ] <!-- open-all-rate-limited-prs -->**Open all rate-limited PRs**
```

Comment:
We recently had some problems with really large Dependency Dashboards not showing all interactive elements:

- #16373

We may run into a similar problem after we merge this PR. Where users with many dependencies can't use the "give me all the PRs" button.

Is there a way to make sure the `Open all $section-name PRs` buttons are always visible and usable?

Reply:
yes @Gabriel-Ladzaretti was working on it.
Gabi do you think such a thing is an option?